src/custom-element.js

```diff
@@ -839,6 +840,12 @@ function createBaseCustomElementClass(win) {
       if (newWidth !== undefined) {
         setStyle(this, 'width', newWidth, 'px');
       }
+      if (opt_newMargins) {
+        setStyle(this, 'marginTop', opt_newMargins.top, 'px');
```

Comment:
I think we should allow each component to be `undefined` and avoid overriding here. This is inline with similar use cases.